File: LocationAlert.Angular/Angular/src/app/_models/weatherpreference.ts
export class WeatherPreference {
  pushHours: number = 1;
  pushMinutes: number = 0;
  alwaysTemp: boolean = false;
  temp: number[] = [-10, 110];
  alwaysRain: boolean = false;
  rain: number[] = [0, 36];
  alwaysSnow: boolean = false;
  snow: number[] = [0, 36];
  alwaysCloud: boolean = false;
  cloud: number[] = [0, 100];
  alwaysWind: boolean = false;
  wind: number[] = [0, 25];
  alwaysHumidity: boolean = false;
  humidity: number[] = [0, 10];
}

File: LocationAlert/LocationAlert.Data.Service/Models/WeatherPreference.cs
namespace LocationAlert.Data.Service.Models
{
    public class WeatherPreference
    {
        public int PushHours { get; set; }
        public int PushMinutes { get; set; }
        public bool AlwaysTemp { get; set; }
        public int[] Temp { get; set; }
        public bool AlwaysRain { get; set; }
        public int[] Rain { get; set; }
        public bool AlwaysSnow { get; set; }
        public int[] Snow { get; set; }
        public bool AlwaysCloud { get; set; }
        public int[] Cloud { get; set; }
        public bool AlwaysWind { get; set; }
        public int[] Wind { get; set; }
        public bool AlwaysHumidity { get; set; }
        public int[] Humidity { get; set; }
    }
}